docs/devguide.md

```diff
@@ -25,7 +25,12 @@ layout:
     │   └── apiserver           # The service catalog API server binary
     │   └── controller-manager  # The service catalog controller manager binary
     ├── contrib                 # Contains all non-essential source
```

Comment:
When I first read `non-essential` source, I though of 'old, stale or wip' code from the service catalog project. It was only at a later time I realized it has sample code and script files for builds among others.